verifier/src/lib.rs

```diff
@@ -79,11 +84,38 @@ pub enum VerificationError {
     },
 }
 
+impl DisplayableError for VerificationError {}
+
+#[derive(Debug, Clone)]
+/// A [`CtOption`] wrapped to implement the [`Display`] trait.
+pub struct DisplayableCtOption<T>(CtOption<T>);
```

Comment:
Not sure if there is a better way to do this. feels a bit clunky to wrap it up to display, perhaps the Verifier should return a `DisplayableCtOption`, or maybe a free function that can print/display `CtOption` would be better :shrug:

Reply:
I'd say return the `CtOption`, but implement `CtOptionDisplay<'_>`, ala [std::path::Display](https://doc.rust-lang.org/std/path/struct.Display.html). From there, you can monkey-patch a `display()` method via a local trait, providing a similar interface to [`Path::display()`](https://doc.rust-lang.org/std/path/struct.Path.html#method.display).

Reply:
Updated to use a trait with `display()` method.